unix_integration/src/daemon.rs

```diff
@@ -660,8 +661,16 @@ async fn main() -> ExitCode {
                 }
             };
 
-            let cl_inner = match CacheLayer::new(
-                cfg.db_path.as_str(), // The sqlite db path
+            let db = match Db::new(cfg.db_path.as_str(), &cfg.tpm_policy) {
+                Ok(db) => db,
+                Err(_e) => {
+                    error!("Failed to create database");
+                    return ExitCode::FAILURE
+                }
+            };
+
+            let cl_inner = match Resolver::new(
```

Comment:
At some point we should probably just pass `db, rsclient, cfg.clone()` and deal with it internally 😄

Reply:
Yes, that's the goal. Sorry there were some emails that @dmulder and I had that you weren't cc'ed on. I'll forward you the goals - docs. But that's the plan :)